docs/getting-started/uwp.rst

```diff
@@ -166,3 +162,33 @@ You can now run the application to see it in action.
 .. image:: uwp/_static/create.png
 
 .. image:: uwp/_static/list.png
+
+Generate runtime directives
+---------------------------
+
```

Comment:
Add a note that this is only needed for deploying... you don't need to do this over and over again during local development